.github/workflows/build.yml

```diff
@@ -12,12 +12,16 @@ jobs:
         with:
           distribution: temurin
           java-version: 11
+      - name: Install and run ipfs
```

Comment:
I just have a tiny concern here.
We are downloading and running external tool before building the lib. In principle, if that gets compromised it could modify the source and we will build\publish something different.
Maybe I'm paranoid. wdyt @Nashatyrev  ?